ksqldb-common/src/main/java/io/confluent/ksql/function/UdfIndex.java

```diff
@@ -142,6 +145,32 @@ void addFunction(final T function) {
     curr.update(function, order);
   }
 
+  T getUdfFunction(final List<SqlArgument> arguments) {
```

Comment:
Created a separate `SqlArgument` track because `TableFunctionFactory` still uses `getFunction` with a `List<SqlType>` passed in.